ansible/roles/software/ckan/catalog/ckan-app/templates/supervisord_gunicorn.conf.j2

```diff
@@ -1,5 +1,5 @@
 [program:catalog-web]
-command=/etc/ckan/server_start.sh -b localhost:5000 --workers=$(( 2 * `cat /proc/cpuinfo | grep 'core id' | wc -l` + 1 )) --proxy-allow-from="127.0.0.1"
+command=/etc/ckan/server_start.sh -b localhost:5000 --workers={{ gunicorn_workers|default(1) }} --proxy-allow-from="127.0.0.1"
```

Comment:
This should go into the role defaults. Since this variable is very similar to [catalog_ckan_wsgi_processes](https://github.com/GSA/datagov-deploy/blob/develop/ansible/roles/software/ckan/catalog/ckan-app/defaults/main.yml#L17), let's rename that variable and use it for both apache-wsgi and for gunicorn.

Reply:
@adborden I remove `gunicorn_workers` and use `catalog_ckan_wsgi_processes` for all. Default value is 2 * CPUs + 1. Removed the production value. All environments will use this default:
https://github.com/GSA/datagov-deploy/pull/2451/commits/c997c236313871b5f0e6bd365e1e6cc419cd241a